docs/docs/guides/sso.mdx

```diff
@@ -5,20 +5,19 @@ title: Configuring Single sign-on
 
 In Infrahub you can configure SSO using either Open ID Connect (OIDC) or can use OAuth2.
 
-The SSO system in Infrahub allows for the configuration of one or more identity providers. While most organizations will only use one provider a reason to have two could be that the providers manage different security domains where one of them might be for regular users the other identity provider could be for administrative accounts.
-Infrahub supports three different OIDC providers:
+We can enable 3 different identity providers in Infrahub:
 
 * PROVIDER1
 * PROVIDER2
 * GOOGLE
 
-All of them work in the same way the main difference is that the one for Google includes some predefined settings that limit the amount of configuration you have to do yourself.
+PROVIDER1 and PROVIDER2 can be used to configure any identity provider that supports OAuth2 or Open ID Connect (OIDC). GOOGLE can be used if you are using Google Workspace as your identity provider, the main difference with the other providers is that GOOGLE has some predefined configuration settings, which makes the configuration more simple.
```

Comment:
We said that we'd avoid using words like "simple" or "easy" to define tasks, as some people might stumble on them anyway.